src/linux/agent/systemui.js

```diff
@@ -56,6 +56,24 @@ rpc.exports = {
       context.startActivity(launchIntent);
     });
   },
+  broadcastAction: function (packageName, receiverName, action) {
+    return performOnJavaVM(function () {
+
+      if (receiverName === null)
+        throw new Error("Please provide receiver name for '"+ packageName + "'");
+
+      if (action === null)
+        throw new Error("Please provide intent action for '" + packageName + "'");
+
+      var intent = Intent.$new();
+      intent.setAction(action);
+
+      var cmp = ComponentName.$new(packageName, receiverName)
```

Comment:
Let's spell this one out to `component`. The current code avoids abbreviations to improve clarity.